core/include/detray/tracks/free_track_parameters.hpp

```diff
@@ -158,7 +159,7 @@ struct free_track_parameters {
     vector_type m_vector = matrix_operator().template zero<e_free_size, 1>();
     covariance_type m_covariance =
         matrix_operator().template zero<e_free_size, e_free_size>();
-    scalar_type m_overstep_tolerance{-1e-4f};
+    scalar_type m_overstep_tolerance{-100 * unit<scalar_type>::um};
```

Comment:
The value seems changed. Did you intend it

Reply:
That was on purpose. This overstepping tolerance worked in many cases pretty well and, if I remember correctly, is also the default value in ACTS

Reply:
ok